### ISSUE TITLE
Chemical dispenser doesn't work when running game in Wine with IE8

### ISSUE THREAD

**Issue by Iamgoofball** [NTStation/NTstation13#164]
_Issue by **[rymd](https://github.com/rymd)** from Tuesday Oct 22, 2013 at 18:49 GMT_
_Originally opened as https://github.com/tgstation/-tg-station/issues/1622_

---

I am running Wine 1.6 in Xubuntu 13.10.

I don't know whether it's IE8 or Wine, but I imagine it's IE8. All that would be required to verify that it is IE8 is to test it by running BYOND on Windows XP, which is not compatible with any version of IE after IE8.

As is visible in the screenshot, none of the buttons on the chemistry dispenser are available.

http://i.imgur.com/Q39bUjP.png

**Comment by Iamgoofball** [comment]
_Comment by **[Mloc](https://github.com/Mloc)** from Tuesday Oct 22, 2013 at 19:08 GMT_

---

This is likely an issue with your environment.

**Comment by Iamgoofball** [comment]
_Comment by **[rymd](https://github.com/rymd)** from Tuesday Oct 22, 2013 at 19:13 GMT_

---

That's possible. Does Nano have any dependencies other than IE and BYOND?

**Comment by Iamgoofball** [comment]
_Comment by **[zekesulastin](https://github.com/zekesulastin)** from Tuesday Oct 22, 2013 at 20:43 GMT_

---

Works fine here with Wine 1.7.4.  In fact, for nano to work at all I had to install ie8 with winetricks alongside the usual BYOND stuff listed in WineHQ, otherwise I got endless script errors.

**Comment by Iamgoofball** [comment]
_Comment by **[KazeEspada](https://github.com/KazeEspada)** from Wednesday Oct 23, 2013 at 07:44 GMT_

---

Nano uses Java, so make sure java is working with wine.

**Comment by Iamgoofball** [comment]
_Comment by **[Giacom](https://github.com/Giacom)** from Wednesday Oct 23, 2013 at 07:54 GMT_

---

You mean javascript.

**Comment by Iamgoofball** [comment]
_Comment by **[rymd](https://github.com/rymd)** from Wednesday Oct 23, 2013 at 21:14 GMT_

---

The same thing happens when using the xmonad window manager (previously I was using xfwm4). I tried testing using Wine + Winetricks rather than PlayOnLinux but I can't seem to get BYOND 500 to work without PlayOnLinux, and the server I am testing on requires BYOND 500. I have installed mfc42, vcrun6, wsh56 and ie8. Not sure what else there is to try. If anyone has any ideas they would like to be tested, it would be appreciated.

I forgot to mention my system is x86-64. That may adversely affect compatibility.
